spec/bowlingFrameSpec.js

```diff
@@ -0,0 +1,79 @@
+describe('Bowling Frame', function() {
+
+  beforeEach(function() {
+    game = new BowlingGame();
+    bowlingFrame = new BowlingFrame(1);
+  });
+
+  it('allows you to enter you first score', function() {
+    bowlingFrame.rollOne(7);
+    expect(game.currentGame[1]).toEqual([7,null]);
+  });
+
+  it('rises an error if you input a score above 10', function() {
+    expect(function() {bowlingFrame.rollOne(11); }).toThrow(new Error('Roll must be between 0 and 10'));
+  });
+
+  it('rises an error if you input a score is below 0', function() {
+    expect(function() {bowlingFrame.rollOne(-1); }).toThrow(new Error('Roll must be between 0 and 10'));
+  });
+
+  it('allows you to enter a second roll score', function() {
+    bowlingFrame.rollOne(7);
+    bowlingFrame.rollTwo(2);
+    expect(game.currentGame[1]).toEqual([7,2]);
+  });
+
+  it('raises an error if second score plus first score is above 10', function() {
```

Comment:
These 5 tests begin `raises an error`, which suggests you might group them within a describe block.
But also try to avoid specifying tests in too "codey" a way, with language such as 'raises an error'. You might phrase it as "doesn't allow" or something like that.